[2023-10-14T03:48:02.879+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: example_pipeline.prepare_data scheduled__2023-10-14T03:47:00+00:00 [queued]>
[2023-10-14T03:48:02.887+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: example_pipeline.prepare_data scheduled__2023-10-14T03:47:00+00:00 [queued]>
[2023-10-14T03:48:02.888+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 1
[2023-10-14T03:48:02.904+0000] {taskinstance.py:1380} INFO - Executing <Task(PythonOperator): prepare_data> on 2023-10-14 03:47:00+00:00
[2023-10-14T03:48:02.911+0000] {standard_task_runner.py:57} INFO - Started process 799 to run task
[2023-10-14T03:48:02.914+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'example_pipeline', 'prepare_data', 'scheduled__2023-10-14T03:47:00+00:00', '--job-id', '212', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_lab.py', '--cfg-path', '/tmp/tmpnrqq14w2']
[2023-10-14T03:48:02.919+0000] {standard_task_runner.py:85} INFO - Job 212: Subtask prepare_data
[2023-10-14T03:48:03.001+0000] {task_command.py:415} INFO - Running <TaskInstance: example_pipeline.prepare_data scheduled__2023-10-14T03:47:00+00:00 [running]> on host 89ffa62682cc
[2023-10-14T03:48:03.343+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='***@example.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='example_pipeline' AIRFLOW_CTX_TASK_ID='prepare_data' AIRFLOW_CTX_EXECUTION_DATE='2023-10-14T03:47:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-10-14T03:47:00+00:00'
[2023-10-14T03:48:03.435+0000] {python.py:194} INFO - Done. Returned value was: None
[2023-10-14T03:48:03.449+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=example_pipeline, task_id=prepare_data, execution_date=20231014T034700, start_date=20231014T034802, end_date=20231014T034803
[2023-10-14T03:48:03.491+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-14T03:48:03.523+0000] {taskinstance.py:2776} INFO - 1 downstream tasks scheduled from follow-on schedule check
